tsdb/wal/watcher.go

```diff
@@ -511,7 +511,7 @@ func (w *Watcher) readSegment(r *LiveReader, segmentNum int, tail bool) error {
 				}
 			}
 			if len(send) > 0 {
-				// Blocks  until the sample is sent to all remote write endpoints or closed (because enqueue blocks).
+				// Blocks  until the sample is sent to the remote write endpoint or closed (because enqueue blocks).
```

Comment:
I think we should actually move this comment to the `WriteTo` interface, the WAL Watcher technically doesn't know/need to know about anything internal to remote write.

```
type WriteTo interface {
        // Append and AppendExemplar should block until the sample is fully accepted by the WriteTo implementation, whether enqueued in memory or successfully written to it's final destination. Once Append/AppendExemplar return the WAL Watcher will not attempt to pass that data to the WriteTo implementation again.
	Append([]record.RefSample) bool
	AppendExemplars([]record.RefExemplar) bool
```

Reply:
it's nice! I will move it.